polyfills/TypedArray/prototype/at/polyfill.js

```diff
@@ -21,7 +21,9 @@
 		return Get(O, ToString(k));
 	}
 
-	if ('__proto__' in self.Int8Array.prototype) {
+	// IE11, and potentially other browsers, have `Int8Array.prototype` inherit directly from `Object.prototype`
```

Comment:
```suggestion
	// in IE11 `Int8Array.prototype` inherits directly from `Object.prototype`
```

let's not say any more than we know